precompiles/werc20/IWERC20.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./../erc20/IERC20MetadataAllowance.sol";
+
+/**
+ * @author Evmos Team
+ * @title Wrapped ERC20 Interface
+ * @dev Interface for representing the native EVM token as ERC20 standard.
+ */
+interface IWERC20 is IERC20MetadataAllowance {
+		/// @dev Emitted when the native tokens are deposited in exchange for the wrapped ERC20.
+    /// @param dst The account for which the deposit is made.
+    /// @param wad The amount of native tokens deposited.
+    event Deposit(address indexed dst, uint wad);
```

Comment:
Nit: use `uint256` for amounts
```suggestion
    event Deposit(address indexed dst, uint256 wad);
```

Reply:
trying to maintain compatibility with the original contract ([here](https://github.com/evmos/wevmos/blob/main/contracts/WEVMOS9.sol)). I'll check if there are differences due to the versions

Reply:
Got it. The abi is using `uint256` anyways